libs/core/src/lib/file-uploader/file-uploader.component.ts

```diff
@@ -174,6 +181,13 @@ export class FileUploaderComponent implements ControlValueAccessor, OnInit, OnDe
         }
     }
 
+    /** @hidden */
+    ngAfterViewInit(): void {
+        if (this.inputRefText) {
+            this._renderer.setStyle(this.inputRefText.nativeElement, 'width', this.width);
```

Comment:
Do we need Renderer2 at all here? We can simply move this to [attr.width]="width" on HTML side. This wasy every consequent change will have effect on UI, not only the first one. Plus reduced complexity

Reply:
What about `[style.width]`? I'm looking for less code here, which works on changes. If it is string and user shoud provide units, why not use `style.width`?